docs/UserGuide.md

```diff
@@ -57,10 +57,13 @@ Example:
 
 ### Delete recipe
 Removes a stored recipe based on the number it is associated with in the `list` <br>
-Usage: `delete <recipe number>` <br>
+Usage: `delete -x <recipe index>` or `delete <recipe name>` <br> 
 Example:
 - delete 1
```

Comment:
I think you forgot to include the index flag.

```suggestion
- delete -x 1
```

Reply:
omg i think my push didnt get updated